src/components/raspberry-pi/button/_index.scss

```diff
@@ -19,6 +20,10 @@
   transition-property: background-color, transform;
   transition-timing-function: transition-timing.$eased;
 
+  @include device.mq('max-width', 420px) {
+    @include size.width(100%);
```

Comment:
@Jonic not sure how you feel about `max-width` media queries but can switch to `min-width` if we prefer.

Reply:
@iamkeir Yeah as a rule of thumb we should avoid `max-width` where possible. It looks innocent enough here, but I don't want people to see `max-width` and think that it's a thing that is good and fine to use. If you could swap this out then that would be SUPER :)